File: frontend/lib/services/firebase_migration_policy.dart
/// Temporary Firebase migration bridge policy.
///
/// IAMINA native authentication is authoritative. Pilot builds keep this off
/// unless a controlled legacy-account migration window is explicitly requested.
const bool kFirebaseMigrationEnabled = bool.fromEnvironment(
  'ENABLE_FIREBASE_MIGRATION',
  defaultValue: false,
);

File: backend/core/firebase_migration_policy.py
"""Fail-closed policy for the temporary Firebase migration bridge.

Firebase is not IAMINA's canonical identity provider. Native Django/IAMINA auth
is authoritative. The migration bridge is disabled unless an operator opts in
explicitly for a controlled legacy-account migration window.
"""
from __future__ import annotations

import os

_TRUTHY = {"1", "true", "yes", "on"}


def firebase_migration_enabled() -> bool:
    """Return True only for an explicit operator opt-in."""
    return os.environ.get("ENABLE_FIREBASE_MIGRATION", "false").strip().lower() in _TRUTHY
